### PR TITLE
Removing duplicates from sorted singly linked list

### PR DESCRIPTION
**Pull request by i-am-aisha**
## Related Issue
#1242 

## Proposed Changes
- Create RemovingDuplicates.cpp
- Update Readme.md

## Additional Info
-Add an algorithm for removing all duplicates elements from a sorted singly linked list.
firstly I take inputs from the user in form of vector elements and then make them nodes of a linked list and then call a function which I made for remove all duplicate elements from that singly linked list.
-Add comments wherever needed
-update README.md accordingly

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

## Output Screenshots
| Screenshot #1      | Screenshot #2   |  Screenshot #2  |
| ----------- | ----------- |----------- |
| Code image 1  | Code image 2    | Code image 3 |
|
Screenshot (344)

Screenshot (345)

Screenshot (346)

Input image
Screenshot (347)

Output image
Screenshot (348)

### CODE REVIEW

File: L-LinkedList/S-SinglyLinkedList/README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
   - [Middle Element of the linked list](#middle-element-of-the-linked-list)
   - [Reverse a linked list](#reverse-a-linked-list)
   - [Searching in a linked list](#searching-in-a-linked-list)
+  - [Removing Duplicates from singly linked list](#removing-duplicates-from-singly-linked-list)
 
 
 # Singly Linked List
```
Comment (Lakhankumawat) [inline L-LinkedList/S-SinglyLinkedList/README.md:18]: Links to other properties like `Algorithms` or `Advantages` , `DisAdv` , `Time Complexity` is missing .

**Review by Lakhankumawat — CHANGES_REQUESTED** [review]

File: L-LinkedList/S-SinglyLinkedList/README.md (hunk 1 above)
Comment (i-am-aisha) [reply L-LinkedList/S-SinglyLinkedList/README.md:18]: Changes updated 👍

**Comment by Lakhankumawat** [comment]
Links on the top , go through this README
https://github.com/Lakhankumawat/LearnCPP/tree/main/S-SortingAlgorithms
image

**Comment by i-am-aisha** [comment]
Changes done 👍

**Review by Lakhankumawat — APPROVED** [review]
LGTM ✨🎉

**Pull request merged**